.github/workflows/tests.yml

```
@@ -47,14 +55,6 @@ jobs:
       - name: Run tests for HTTP Mode adapters (Falcon 3.x)
```

Can you remove 3.x from this label? The latest version is now 4.x.
```suggestion
      - name: Run tests for HTTP Mode adapters (Falcon)
```

---

Good catch 💯